react_juce/core/CanvasView.h

```diff
@@ -9,739 +9,74 @@
 
 #pragma once
 
-#include "View.h"
+#include"View.h"
```

Comment:
Weird space? Same thing, let's not block the merge for it. Clang-format will save us!